tests/src/JIT/Performance/CodeQuality/V8/DeltaBlue/DeltaBlue.cs

```diff
@@ -906,17 +906,22 @@ public static void Bench()
         }
     }
 
-    public void inst_main(String[] args)
+    public bool inst_main(String[] args)
     {
         int iterations = 200; // read iterations from arguments, walter 7/97
-        try
-        {   // read iterations from arguments, walter 7/97
-            iterations = Int32.Parse(args[0]);
-        }
-        catch (Exception)
+        if (args.Length > 0)
         {
+            bool parsed = Int32.TryParse(args[0], out iterations);
+            if (! parsed)
```

Comment:
`! parsed` looks a little funny (as opposed to `!parsed`), don't know if that was intentional.